marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
No circular dependencies should exist: compat.py should not import anything from utils.py (the reverse is ok). And, yes, those functions are pure python stuff.